cookbook/examples/streamlit/medical_imaging/README.md

```diff
@@ -0,0 +1,28 @@
+# Medical Imaging Diagnosis Agent
+
+Medical Imaging Diagnosis Agent is a medical imaging analysis agent that analyzes medical images and provides detailed findings by utilizing models and external tools: DuckDuckGo
+
+### 1. Create a virtual environment
+
+```shell
+python3 -m venv ~/.venvs/aienv
+source ~/.venvs/aienv/bin/activate
+```
```

Comment:
```suggestion
```shell
./scripts/cookbook_setup.py
source ./agnoenv/bin/activate
```
```

We can make this change for all the streamlit apps.